autopilot/CMakeLists.txt

```diff
@@ -8,15 +8,14 @@ add_compile_options(-std=c++11)
 find_package(catkin REQUIRED COMPONENTS
   auv_interfaces
   fin_control
-  geodesy

```

Comment:
@patapufate hmm, removing waypoint following?

Reply:
Good catch. Restored in 8261d61

Reply:
I think we'll need to revert more than that -- I see lots of changes to the `autopilot`.